pkg/collector/corechecks/snmp/internal/checkconfig/config_metric_test.go

```diff
@@ -43,3 +43,54 @@ func Test_normalizeRegexReplaceValue(t *testing.T) {
 		})
 	}
 }
+
+func Test_getMappedValue(t *testing.T) {
+	tests := []struct {
+		val                 string
+		mapping             map[string]string
+		expectedMappedValue string
+		expectedError       string
+	}{
+		{
+			"1",
```

Comment:
Could you add a descriptive `name` to the struct that will be used in `t.Run(tt.name, ...`, this will add readability and some documentation for each case.

Example name "one value" (first case), "multiple value" (second case), "empty mapping" (last case, 4).

Example: https://github.com/DataDog/datadog-agent/blob/b643d89082b6490af64812209eddaaa9b016e306/pkg/collector/corechecks/snmp/internal/report/report_metrics_test.go#L372

This will help readability and document what each case does.

Reply:
Very true, thanks!